tests/testthat/test-reporter-junit.R

```diff
@@ -0,0 +1,6 @@
+context("JUnitReporter")
+
+test_that("Junit reporter produces XML", {
+
```

Comment:
Sure, I can help out. I don't have a ton of free time. I'll look at what has happened since I last checked this PR tonight and see what I can do.